pkg/cmd/stack/translate.go

```diff
@@ -459,6 +477,54 @@ func getAddPermissionsInitContainer(svcName string, svc *model.Service) apiv1.Co
 	return initContainer
 }
 
+func getWaitForSvcsInitContainer(svcName string, s *model.Stack) *apiv1.Container {
+	command := "echo waiting for dependent services..."
+	for dependentSvc, condition := range s.Services[svcName].DependsOn {
+		if condition.Condition == model.DependsOnServiceHealthy {
+			if len(s.Services[dependentSvc].Ports) == 0 {
+				condition.Condition = model.DependsOnServiceRunning
+			}
+		}
+		command += getDependentCommand(condition, dependentSvc, s)
+	}
+
+	if command != "echo waiting for dependent services..." {
+		initContainer := &apiv1.Container{
+			Name:    "wait-for-svcs",
+			Image:   "bitnami/kubectl",
```

Comment:
I think it is better if we build our own image with a small go program that does this logic.